clients/src/main/java/org/apache/kafka/common/network/SslTransportLayer.java

```diff
@@ -352,8 +352,12 @@ private void handshakeFinished() throws IOException {
             //remove OP_WRITE if we are complete, otherwise we still have data to write
             if (!handshakeComplete)
                 key.interestOps(key.interestOps() | SelectionKey.OP_WRITE);
-            else
+            else {
                 key.interestOps(key.interestOps() & ~SelectionKey.OP_WRITE);
+                SSLSession session = sslEngine.getSession();
+                log.trace("SSL handshake completed successfully with peerHost '{}' peerPort {} peerPrincipal '{}' cipherSuite '{}'",
```

Comment:
Should this one be at `DEBUG` level? We have a debug log for when connections are established in `NetworkClient` and this seems similar. What do you think?